widget/class-widget.php

```diff
@@ -138,7 +156,7 @@ public function widget( $args, $instance ) {
    public function form( $instance ) {
 
       // Default title to 'Volunteer Opportunities'
-      $title = ! empty( $instance['title'] ) ? $instance['title'] : __( 'Volunteer Opportunities', 'wired-impact-volunteer-management' );
+      $title = ! empty( $instance['title'] ) ? $instance['title'] : __( '', 'wired-impact-volunteer-management' );
```

Comment:
It won't let me comment on the line of code, but can we check the box to show when the opportunity takes place by default for a new widget? I think it makes sense to show it in more cases than not.